com.github.Anuken.Mindustry.yml

```diff
@@ -7,7 +7,9 @@ sdk-extensions:
 
 command: mindustry.sh
 finish-args:
+  - --socket=wayland
```

Comment:
Have you checked if this works?

Reply:
Hmm... you're right, xeyses still works for Mindustry.

Im gonna have to debug this further...